charts/k8ssandra/templates/prometheus/service_monitor.yaml

```diff
@@ -1,8 +1,13 @@
+{{- $prometheus := .Values.monitoring.prometheus -}}
+
+{{- if and $prometheus.provision_service_monitors }}
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ .Release.Name }}-prometheus-k8ssandra
   labels:
+    heritage: Helm
+    release: k8ssandra
```

Comment:
What are these? What release?

Reply:
That's a mistake, testing removal of `heritage: Helm` now. `release` should be `release: {{ .Release.Name }}`. This is used by the Prometheus chart to pick up service monitors that match the release.

Reply:
This has been resolved in the latest push.